packages/angular/projects/ui-angular/package.json

```diff
@@ -14,7 +14,7 @@
     "directory": "packages/angular/projects/ui-angular"
   },
   "peerDependencies": {
-    "aws-amplify": "next"
+    "aws-amplify": "5.x.x"
```

Comment:
I assume `5.x.x` === `^5.0.0` right? Does it matter if we mix and match the 2 syntaxes

Reply:
Meant to note that. Before we started pointing to `aws-amplify/js@next` our peerDeps were defined as the range of `3.x.x || 4.x.x`, this was meant to continue that pattern. The explicit other `aws-amplify` deps in this PR are traditionally pointed to `aws-amplify/js@latest`, but since `latest` is still v4 added the explicit `5.0.0`